#pose to be executed by move_base_client(robot_name)
string robot_name
geometry_msgs/Pose2D pose
